lib/ioki/http_adapter.rb

```diff
@@ -10,20 +10,20 @@ def self.get(config)
       ::Faraday.new(config.api_base_url, headers: headers(config)) do |f|
         f.adapter :net_http
 
+        f.request :authorization, 'Bearer', -> { config.api_token }
+        f.request :json # encode req bodies as JSON
```

Comment:
Why is this back on top again? Doesn't this revert the changes of #44?

Reply:
These two are the middleware for the outgoing `request`. The order of the middleware for the incoming `response` is what I changed in #44. The thing is, that the logger inspects both `resquest` and `response` even if it is only called on the response (don't ask me why) and I noticed, that the authorization header would not be logged if it is configured in this setup after the logger. I was not able to test yet how this order affects the logging of the outgoing request bodies, such as a POST request body.

I can take a look after the weekend though.